packages/backend/src/plugins/kubernetes.ts

```diff
@@ -10,6 +10,7 @@ export default async function createPlugin(
   const { router } = await KubernetesBuilder.createBuilder({
     logger: env.logger,
     config: env.config,
+    permissions: env.permissions,
```

Comment:
This change should also be on the `packages/backend/src/plugins/scaffolder.ts` file.